tests/generic_config_updater/test_ecn_config_update.py

```diff
@@ -54,13 +54,27 @@ def ensure_application_of_updated_config(duthost, configdb_field, values):
         values: expected value(s) of configdb_field
     """
     def _confirm_value_in_asic_db():
-        table_name = duthost.shell('sonic-db-cli ASIC_DB keys *WRED*')["stdout"]
-        wred_data = duthost.shell('sonic-db-cli ASIC_DB hgetall {}'.format(table_name))["stdout"]
-        wred_data = ast.literal_eval(wred_data)
-        for field, value in zip(configdb_field.split(','), values.split(',')):
-            if value != wred_data[WRED_MAPPING[field]]:
-                return False
-        return True
+        if(duthost.facts['asic_type'] == 'cisco-8000'):
```

Comment:
We don't want to see any more platform-specific changes. Could you do some enhancements for the fix? such as you can get the default WRED before the test, and exclude the default oid during the test.

Reply:
Created another PR (https://github.com/sonic-net/sonic-mgmt/pull/9610) to remove the platform specific changes.
The new code should work for cisco as well as other vendors